README.md

```diff
@@ -1,71 +1,33 @@
+[![npm downloads](https://img.shields.io/npm/dm/@mui/material.svg)](https://www.npmjs.com/package/@mui/base)
+[![CircleCI](https://circleci.com/gh/mui/material-ui/tree/master.svg?style=shield)](https://app.circleci.com/pipelines/github/mui/base-ui?branch=master)
+[![Coverage Status](https://img.shields.io/codecov/c/github/mui/material-ui/master.svg)](https://codecov.io/gh/mui/base-ui/branch/master)
 [![Follow on X](https://img.shields.io/twitter/follow/MUI_hq.svg?label=follow+MUI)](https://twitter.com/MUI_hq)
-[![Renovate status](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://github.com/mui/material-ui/issues/27062)
-[![Average time to resolve an issue](https://isitmaintained.com/badge/resolution/mui/material-ui.svg)](https://isitmaintained.com/project/mui/material-ui 'Average time to resolve an issue')
+
+<!-- [![Renovate status](https://img.shields.io/badge/renovate-enabled-brightgreen.svg)](https://github.com/mui/material-ui/issues/27062) -->
+<!-- [![Average time to resolve an issue](https://isitmaintained.com/badge/resolution/mui/material-ui.svg)](https://isitmaintained.com/project/mui/material-ui 'Average time to resolve an issue') -->
+
 [![Open Collective backers and sponsors](https://img.shields.io/opencollective/all/mui-org)](https://opencollective.com/mui-org)
 [![CII Best Practices](https://bestpractices.coreinfrastructure.org/projects/1320/badge)](https://bestpractices.coreinfrastructure.org/projects/1320)
 
 </div>
 
 ## Documentation
 
-### Material UI
-
-Visit [https://mui.com/material-ui/](https://mui.com/material-ui/) to view the full documentation.
-
-<details>
-  <summary>Older versions</summary>
-
-- **[v4.x](https://v4.mui.com/)** ([Migration from v4 to v5](https://mui.com/material-ui/migration/migration-v4/))
-- **[v3.x](https://v3.mui.com/)** ([Migration from v3 to v4](https://mui.com/material-ui/migration/migration-v3/))
-- **[v0.x](https://v0.mui.com/)** ([Migration to v1](https://mui.com/material-ui/migration/migration-v0x/))
-
-</details>
-
-**Note:** `@next` only points to pre-releases.
-Use `@latest` for the latest stable release.
-
-### Joy UI
-
-Visit [https://mui.com/joy-ui/getting-started/](https://mui.com/joy-ui/getting-started/) to view the full documentation.
-
-**Note**: Joy UI is still in beta.
-We are adding new components regularly and you're welcome to contribute!
-
-### Base UI
-
 Visit [https://mui.com/base-ui/](https://mui.com/base-ui/) to view the full documentation.
 
-**Note**: Base UI is still in beta.
-We are adding new components regularly and you're welcome to contribute!
-
-### MUI System
-
-Visit [https://mui.com/system/getting-started/](https://mui.com/system/getting-started/) to view the full documentation.
+**Note**: Base UI is currently being rewritten and isn't adding new features or components to the current implementation.
+More details about future plans for it can be found in this blog post.
```

Comment:
This callout (with the revised wording) will also be available in the documentation like we talked 👌